e2e/testconfig/testconfig.go

```diff
@@ -150,3 +160,53 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string) ibc.Cha
 		NoHostMount:    false,
 	}
 }
+
+// defaultModifyGenesis will only modify governance params to ensure the voting period and minimum deposit
+// are functional for e2e testing purposes.
+func defaultModifyGenesis(denom string) func([]byte) ([]byte, error) {
+	return func(genbz []byte) ([]byte, error) {
+		genDoc, err := tmtypes.GenesisDocFromJSON(genbz)
+		if err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis bytes into genesis doc: %w", err)
+		}
+
+		var appState genutiltypes.AppMap
+		if err := json.Unmarshal(genDoc.AppState, &appState); err != nil {
+			return nil, fmt.Errorf("failed to unmarshal genesis bytes into app state: %w", err)
+		}
+
+		cfg := simappparams.MakeTestEncodingConfig()
+		govv1beta1.RegisterInterfaces(cfg.InterfaceRegistry)
+		cdc := codec.NewProtoCodec(cfg.InterfaceRegistry)
```

Comment:
what do you think about moving this to some separate initialization function, is there a benefit to having a new instance each time?

Reply:
No benefit. Ideally this function would be exposed on [ibctest](https://github.com/strangelove-ventures/ibctest/issues/260). Where would you suggest we add this initialization function? Are you suggesting we keep a global variable? I think that'd be fine since it is just for testing

I'd be happy to add a `generateCodec()` function, but it would still recreate the instance each time

Reply:
Yeah I guess there wouldn't be much benefit either way, maybe we could consider it if we find ourselves generating the same codec in multiple places. It's probably fine for now to leave this as is

Reply:
I'll leave as is. I think it'll be easier to wait for a natural place to initialize the codec than to look for one right now